examples/serial_shallow_water_turbulence.jl

```diff
@@ -0,0 +1,80 @@
+using Statistics
```

Comment:
Should we cut this example as it's just here as a sanity check to compare with the serial case?

Reply:
Done. Yeah I agree the MPI example is the more interesting one.